sanic/app.py

```diff
@@ -570,9 +570,10 @@ def run(self, host="127.0.0.1", port=8000, debug=False, before_start=None,
                 serve(**server_settings)
             else:
                 serve_multiple(server_settings, workers)
-        except:
+        except Exception:
```

Comment:
Not really, a bare `except:` captured everything that also inherits `BaseException` including signals like KeyboardInterrupt or SystemExit.
that you probably don't want to capture here. it is considered anti-pattern.
Your call though.